source/_posts/2025-03-05-release-20253.markdown

```diff
@@ -0,0 +1,563 @@
+and we have made it even better!
+
+### Knowing which tile card can be controlled
+
+Tile cards are awesome! It is arguably the best and most versatile dashboard
+card Home Assistant current has. But ever since its creation, it had a little
+flaw that you might have ran into yourself a few time...
+
+How would you know if tapping a tile card would trigger an action or when
+it would just display the current state value of a sensor? We have subtly fixed
+these things in this release!
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-shapes.png" />
+
+When there is no action (or only a more info attached), we don't show the circle
+when it does have an actions, it will show the circle, meaning you can tap that
+little button.
+
+### Position tile card features
+
+Ever since we added support for resizing cards, we gained the ability to make
+very wide tile cards. These cards would have a lot of empty space surrounding
+it, which, well at times could be helpful I guess.
+
+But what if we could do something useful with that space? Now you can!
+All features of a tile card, can now be positioned in-line in the tile card!
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-inline-feature.png" />
+
+This brings a completely new look and feel option to the tile car, of which we
+are sure, you are going to love! From the setting perspective, you can thus
+set the position of the feature to be either vertically or on the right side.
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-inline-settings.png" />
+
+Note, that only the first feature can be positioned in-line, the rest will
+be positioned below the tile card.
+
+### New tile card features
+
+This release also ships with two new tile card features, that can be
+added to your entities to provide new controls. These have been, contributed by
+[@jpbede], so a big shout out to him!
+
+[@jpbede]: https://github.com/jpbede
+
+#### Switch toggle
+
+The first addition, is one that is just surprising we didn't have it before.
+A switch toggle! This allows you to toggle a switch entity directly from the
+tile card.
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-switch-toggle.png" />
+
+#### Counter actions
+
+The second addition, is a counter toggle. This allows you to increase or
+decrease a counter entity directly from the tile card.
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-counter-actions.png" />
+
+### Tiny interaction improvements
+
+More tweaks to the tile card have been made to improve the interaction
+experience with them.
+
+For example, we now have nice tiny little animations when you however over
+the tile card, making it more clear that it is interactive; but also when
+you tap the tile card, it will now show a little animation to indicate that
+the tap was registered.
```

Comment:
```suggestion
For example, we now have nice tiny little animations when you hover over
the tile card, making it more clear that it is interactive; but also when
you tap the tile card, it will now show a little animation to indicate that
the tap was registered.
```